sessionctx/variable/tidb_vars.go

```diff
@@ -741,6 +741,10 @@ const (
 	// TiDBMaxAutoAnalyzeTime is the max time that auto analyze can run. If auto analyze runs longer than the value, it
 	// will be killed. 0 indicates that there is no time limit.
 	TiDBMaxAutoAnalyzeTime = "tidb_max_auto_analyze_time"
+	// TiDBFastDDL indicates whether use lighting to help acceleate adding index stmt.
+	TiDBFastDDL = "tidb_fast_ddl"
```

Comment:
It is recommend to use `tidb_enable_fast_ddl` to make it consistent with other names.

Reply:
I will change it to tidb_enable_fast_ddl.